codes/qubits/steane.yml

```diff
@@ -31,7 +31,8 @@ features:
   # decoders: 
     - ''
 
-  transversal_gates: 'Clifford gates are transversal'
+  transversal_gates: 
+    - 'Clifford gates are transversal'
```

Comment:
It inherits this from CSS codes, so put that comment there. We may have a way to tack on features from parent codes.